src/twisted/runner/test/test_procmon.py

```diff
@@ -94,6 +96,9 @@ class DummyProcessReactor(MemoryReactor, Clock):
     @ivar spawnedProcesses: a list that keeps track of the fake process
         instances built by C{spawnProcess}.
     @type spawnedProcesses: C{list}
+
+    @ivar spawnProcessException: An exception which spawnProcess() will raise.
+    @type spawnProcessException: C{Exception}
```

Comment:
This `@type` should be a type annotation; also, it's not correct, because it's `Exception | None`.